doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -165,24 +169,24 @@ which we see the output of ``ceph tell mon.c mon_status``::
                 "name": "c",
                 "addr": "127.0.0.1:6795\/0"}]}}
 
-This output reports that there are three monitors in the monmap (*a*, *b*, and
-*c*), that quorum is formed by only two monitors, and that *c* is in quorum as
-a *peon*.
+This output reports that there are three monitors in the monmap (``a``, ``b``,
+and ``c``), that quorum is formed by only two monitors, and that ``c`` is in
+quorum as a ``peon``.
```

Comment:
I would say that c is a peon, I think it’s confusing to say it’s in quorum as a peon.

Reply:
I'll do this in a near-future PR.

Reply:
https://github.com/ceph/ceph/pull/58156